funding/manager.go

```diff
@@ -3259,6 +3259,16 @@ func (f *Manager) handleInitFundingMsg(msg *InitFundingMsg) {
 	if chanReserve == 0 {
 		chanReserve = f.cfg.RequiredRemoteChanReserve(capacity, ourDustLimit)
 	}
+	if chanReserve < ourDustLimit {
```

Comment:
We already have a similar check in `CommitConstraints`, though that doesn't check against the capacity.